contracts/beacon/Voucher.sol

```diff
@@ -432,11 +434,18 @@ contract Voucher is Initializable, IVoucher {
         if (sponsoredAmount != dealPrice) {
             // Transfer non-sponsored amount from the iExec account of the
             // requester to the iExec account of the voucher
-            IERC20(iexecPoco).transferFrom(
-                requestOrder.requester,
-                address(this),
-                dealPrice - sponsoredAmount
-            );
+            //slither-disable-start arbitrary-send-erc20
```

Comment:
Maybe add note like "we can disable since request order is signed by requester (so is ok to pay for) & caller is onlyAuthorized"